test/integration/functional_test.go

```diff
@@ -299,9 +299,10 @@ func validateImageCommands(ctx context.Context, t *testing.T, profile string) {
 	// docs: Make sure image building works by `minikube image build`
 	t.Run("ImageBuild", func(t *testing.T) {
 		MaybeParallel(t)
-
-		if _, err := Run(t, exec.CommandContext(ctx, Target(), "-p", profile, "ssh", "pgrep", "buildkitd")); err == nil {
-			t.Errorf("buildkitd process is running, should not be running until `minikube image build` is ran")
+		if runtime.GOOS != "windows" {
```

Comment:
I think there might have been some confusion on my last comment. We should check do this check on Windows as well, so the if statement should be removed.

What I was trying to say is that on line 311, if not on Windows the path to the Dockerfile should continue being `filepath.Join(*testdataDir, "build")`, but on Windows we should use `"testdata\\build"` to test this change.

Reply:
Actually I found that on Windows, the result of filepath.Join(*testdataDir, "build") is exactly "testdata\\build", so actually we don't need to modify this part, or write a seperate test.

Reply:
Ah, gotcha, thanks for the info